internal/module/kernelmapper.go

```diff
@@ -51,15 +51,17 @@ type kernelMapperHelperAPI interface {
 	findKernelMapping(mappings []kmmv1beta1.KernelMapping, kernelVersion string) (*kmmv1beta1.KernelMapping, error)
 	prepareModuleLoaderData(mapping *kmmv1beta1.KernelMapping, mod *kmmv1beta1.Module, kernelVersion string) (*api.ModuleLoaderData, error)
 	replaceTemplates(mld *api.ModuleLoaderData) error
+	GetRelevantBuild(moduleBuild *kmmv1beta1.Build, mappingBuild *kmmv1beta1.Build) *kmmv1beta1.Build
+	GetRelevantSign(moduleSign *kmmv1beta1.Sign, mappingSign *kmmv1beta1.Sign, kernel string) (*kmmv1beta1.Sign, error)
```

Comment:
I think we can unexport those methods. No other package is using them (excluding the build package which uses the other methods from the build package.